client/exercises/links-and-buttons/jump-links/README.md

```diff
@@ -0,0 +1,9 @@
+### Jump links
+
+Ne-ar plăcea să construiești ceva ce există chiar în cadrul lecției și anume acele jump links. De ce ne dorim asta? Pentru că în practică va trebui să folosești astfel de link-uri. Pentru pasul ăsta nu îți fă griji dacă nu vei obține acel efect de **“smooth scrolling”**, o să vorbim de el la partea de CSS. Concentrează-te doar pe conceptele prezentate până aici și scrie codul pentru button element, nu mai mult.
+
+![jump-links-todos](./assets/jump-links-demo.jpg)
```

Comment:
I am confused by this exercise. Should I implement everything in the picture? I looked at your solution, it seems you did not 🤔

My suggestion is to:

1. simplify the picture -> there's a lot going on and it might be overwhelming for some (by simplify I mean drop the bullets in the menu, keep just 2-3 items in the menu, and also simplify the right part, less text)
2. The demo in README should be a Video/Gif - not a picture. We want the people to understand the page should scroll when clicking the links
3. Update the demo with the new code

Remember to update the demo with the new code